src/components/posts/PostItem.js

```diff
@@ -15,18 +22,20 @@ function PostItem(props) {
           />
         </div>
         <div className={classes.content}>
-            <div className={classes.top}>
-                <h3>{props.author}</h3>
-                <h4>{props.date}</h4>
-            </div>
+          <div className={classes.top}>
+            <h3>{props.author}</h3>
+            <h4>{props.date}</h4>
+          </div>
           <h5>{props.content}</h5>
         </div>
         <div className={classes.actions}>
           <div className={classes.like}>
-            <button>like</button>
+            
+            <button onClick={likeHandler}>like</button>
```

Comment:
For future work when you add in this like button it might be interesting to see if you can try and reuse the `DeleteButton` component (though `DeleteButton` might need a name change in that case)

Reply:
I will try that